gamemodes/terrortown/gamemode/client/cl_player_ext.lua

```diff
@@ -184,3 +184,75 @@ function GM:SetupMove(ply, mv, cmd)
 
 	hook.Run("TTT2PlayerReady", ply)
 end
+
+---
+-- Sets a revival reason that is displayed in the revival HUD element.
+-- It supports a language identifier for translated strings.
+-- @param [default=nil]string name The text or the language identifer, nil to reset
```

Comment:
wrong doc style

Reply:
same here, what's wrong with it?

Reply:
Already told it so many times :D `@param[default=nil] string` not `@param [default=nil]string`

Reply:
I just checked the code and both styles are used 50:50. I guess one of them is wrong.

Reply:
They are used bc you submitted them i guess :')

Reply:
Just checked it, most of them are by me, but not all of them. And you did not notice it while reviewing :P